h2o-py/tests/testdir_algos/gam/pyunit_PUBDEV_7709_gam_error_messages.py

```diff
@@ -0,0 +1,45 @@
+from __future__ import division
+from __future__ import print_function
+from past.utils import old_div
+import sys
+sys.path.insert(1, "../../../")
+import h2o
+from tests import pyunit_utils
+from h2o.estimators.gam import H2OGeneralizedAdditiveEstimator
+
+# In this test, we want to make sure the correct error message is returned when a gam column has two
+# few values to be divided into bins and when the knot locations are not increasing in value.
+def test_gam_knots_key():
+    # bad gam_column with not enough values is chosen to be gam_column
+    h2o_data = h2o.import_file(path=pyunit_utils.locate("smalldata/prostate/prostate_complete.csv.zip"))
+    h2o_data.head()
+    myY = "CAPSULE"
+    myX = ["ID","AGE","RACE","GLEASON","DCAPS","PSA","VOL","DPROS"]
```

Comment:
can gen be a gam column ("GLEASON") also one of the feature columns?

Reply:
yes.